### ISSUE TITLE
Unable to connect

### ISSUE THREAD

**Issue by MaikKlein** [GPUOpen-Tools/Radeon-GPUProfiler#4]
When I try to connect to `localhost 27300`, I get `Operating in Headless mode` and it fails to connect.
```
[RDP] Loaded RDP settings file.
[RDP] Attempting to start Radeon Developer Service locally.
[RDP] Launching local RDS.
[RDS] Failed to connect RDP to RDS. Connection timed out.
```

Ubuntu 16.04.2 x64 and 480rx with 17.30

**Comment by john-amd** [comment]
Hi MaikKlein,

Thanks for your report. We'll look into this issue, and are hoping to have a fix soon.

**Comment by ahosier** [comment]
Hi MaikKlein,

We have tried to duplicate the issue you're seeing and have been unsuccessful so far. The current Linux driver doesn't support 16.04.2 and only supports 16.04.3. You may have a situation where the driver looks like it is installed but some functionality may be missing.

Recommend you upgrade Ubuntu to 16.04.3 and then install the latest linux driver from here:
http://support.amd.com/en-us/kb-articles/Pages/AMDGPU-PRO-Driver-for-Linux-Release-Notes.aspx Scroll down to the link:
AMDGPU-Pro Driver Version 17.30 for Ubuntu 16.04.3​
and download and install this driver and see if that helps. Please get back in touch if you're still seeing issues.

**Comment by MaikKlein** [comment]
I upgraded to 16.04.3 and installed the driver but it still fails for me. Also if I just do `./RadeonDeveloperService`

![](https://i.imgur.com/cduC09V.png)

Is that expected?

**Comment by ahosier** [comment]
Currently RGP just supports Ubuntu 16.04.3 out of the box (just the unity desktop). We haven't tried running it with other desktop environments, but obviously there is an issue with these. That being said, you should still be able to get a trace. It just means that the service is running without the system tray icon, usually shown in the top bar. Under the hood, it's using Qt to query if the tray icon is available using QSystemTrayIcon::isSystemTrayAvailable() and this is failing for some reason.

**Comment by ahosier** [comment]
Hi MaikKlein,

We have just released a new version of Radeon GPU Profiler (V1.03) which should allow you to capture profiles in headless mode. Please get back to us if you still see issues.

**Comment by MaikKlein** [comment]
@ahosier Thanks, I am now able to connect. Unfortunately, I currently have RADV installed and therefor not save profiles. I think that is expected?

I think this issue can be closed.
